FoodOrderingSystem.py

```diff
@@ -0,0 +1,925 @@
+    4. Cancel order
+    5. View order status
+                
+    Select option: 
+
+    """
+    order = int(input(selection))
+
+    while order != 0:
+
+        if order == CUST_OPT_PROCESS_ORDER:
+            cust_id = input("Enter customer ID: ")
+            fos.process_order(cust_id)
+
+        elif order == CUST_OPT_VIEW_ORDER:
+            order_id = input("Enter order ID: ")
+            fos.view_order(order_id)
+                
+        elif order == CUST_OPT_CHECKOUT:
+            order_id = input("Enter order ID: ")
+            order_address = input("Enter delivery address: ")
+            checkout = int(input("Press 1 to confirm checkout: "))
+            if checkout == 1:
+                order_status = "Checkedout"
+            checkout_time = datetime.now()
+            delivery_time = timedelta(minutes=30)
+            estimated_time = checkout_time + delivery_time
+            session = fos.Session()
+            view_grand_total = fos.customer.view_order_grand_total(order_id, session)
+            if view_grand_total:
+                for fd, cos, cd, cosa, t in view_grand_total:
+                    bill_amount = t
+            fos.checkout(order_id, order_status, order_address, checkout_time, estimated_time, bill_amount)
+
+        elif order == CUST_OPT_CANCEL_ORDER:
+            order_id = input("Enter order ID: ")
+            cancel = int(input("Press 1 to confirm cancellation: "))
+            if cancel == 1:
+                order_status = "Cancelled"
+            fos.cancel_order(order_id, order_status) 
+
+        elif order == CUST_OPT_VIEW_ORDER_STATUS:
+            order_id = input("Enter order ID: ")
+            fos.view_orders_status(order_id)
+
+        order = int(input(selection))               
+
+def process_customer_options_flow(fos):
+    cust_options = """ 
+        
+    Welcome to the employee interface! Please press the below options:
+        
+    0. Quit
+    1. View menu
+    2. Customer signup
+    3. Customer login
+
+    Your Option:
+
+    """        
+        
+    customer_options = int(input(cust_options))
+
+    while customer_options != 0:
+        
+        if customer_options == CUST_OPT_VIEW_MENU:
+            fos.view_menu()
+
+        elif customer_options == CUST_OPT_CUSTOMER_SIGNUP:
+            cust_name = input("Enter customer name: ")
+            cust_phone = input("Enter customer phone number: ")
+            cust_email = input("Enter customer email address: ")
+            fos.customer_signup(cust_name, cust_phone, cust_email)
+
+        elif customer_options == CUST_OPT_CUSTOMER_LOGIN:
+            cust_id = input("Enter customer ID: ")
+            c = fos.customer_login(cust_id)
+            if c:
+                print("Login successful! \nWelcome {} \nCustomer ID {}".format(c.cust_name, c.cust_id))
+            else:
+                print("Login not sucessfully. Please try again or signup!")
+                break
+            process_order_flow(fos)
```

Comment:
This part is confusing, why do you put `break` here? and why you're calling the `process_order_flow(fos)` here?